collections/datasets/index.php

```diff
@@ -97,9 +100,9 @@ function validateAddForm(f){
 	include($SERVER_ROOT.'/includes/header.php');
 	?>
 	<div class='navpath'>
-		<a href='../../index.php'>Home</a> &gt;&gt;
-		<a href="../../profile/viewprofile.php?tabindex=1">My Profile</a> &gt;&gt;
-		<a href="index.php"><b>Dataset Listing</b></a>
+		<a href='../../index.php'> <?php echo (isset($LANG['HOME'])?$LANG['HOME']:'Home') ?> </a> &gt;&gt;
```

Comment:
Add spaces for ternary operator syntax:
`foo ? bar : baz` rather than `foo?bar:baz`